pkg/inject/report.go

```diff
@@ -0,0 +1,32 @@
+package inject
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Report contains the Kind and Name for a given workload along with booleans
+// describing the result of the injection transformation
+type Report struct {
```

Comment:
Is there a way to consolidate the report here and the `generateReport()` in the CLI? Do they serve different purposes?

Reply:
`Report` captures the common functionality leveraged by the implementations of `generateReport()` that are different for cli inject and cli uninject. That's why I put `Report` in the lib, and the `generateReport()` functions in their respective cli files.